src/applications/post-911-gib-status/tests/components/PrintPage.unit.spec.jsx

```diff
@@ -15,23 +15,26 @@ describe('<PrintPage/>', () => {
   afterEach(() => pushSpy.reset());
 
   it('should render', () => {
-    // eslint-disable-next-line va-enzyme/unmount
+    // Not necessary if not componentWillUnmount
+    // eslint-disable-next-line va/enzyme-unmount
     const wrapper = shallow(<PrintPage {...defaultProps} />, {
       disableLifecycleMethods: true,
     });
     expect(wrapper.type()).to.equal('div');
   });
 
   it('renders a UserInfoSection child', () => {
-    // eslint-disable-next-line va-enzyme/unmount
+    // Not necessary if not componentWillUnmount
+    // eslint-disable-next-line va/enzyme-unmount
```

Comment:
Would it work to put a `wrapper.unmount()` after the expect?

Reply:
No, I tried that in all of them. keep getting the same error:
```
<PrintPage/> should render:
     TypeError: Cannot read property 'classList' of null
```

Reply:
I guess it makes sense since this particular test disables the lifecycle methods everywhere. I wonder why they're disabled, though. Do they still pass if you remove that option and unmount after?

Also, is it the same story for the other specs that are not for `PrintPage`?

Reply:
yes, it's the same story. I fixed the ones that didn't fail